ui/decredmaterial/editor.go

```diff
@@ -24,24 +25,38 @@ type Editor struct {
 	LineColor  color.NRGBA
 
 	flexWidth float32
-	//IsVisible if true, displays the paste and clear button.
+	//IsVisible if true, displays the clear button.
 	IsVisible bool
 	//IsRequired if true, displays a required field text at the buttom of the editor.
 	IsRequired bool
 	//IsTitleLabel if true makes the title label visible.
 	IsTitleLabel bool
 	//Bordered if true makes the adds a border around the editor.
 	Bordered bool
+	//IsPassword if true, displays the show and hide button.
+	IsPassword bool
 
 	requiredErrorText string
 
-	pasteBtnMaterial IconButton
-	clearBtMaterial  IconButton
+	clearBtMaterial    IconButton
+	showPasswordButton IconButton
+
+	showHidePassword *widget.Clickable
+	iconhide         *widget.Image
```

Comment:
This button is also available in iconvg format. The hide and show buttons have the names ActionVisibility & ActionVisibilityOff. Before using a png format for any icon check [here](https://material.io/resources/icons/?style=baseline) for the icon first. If you can find it there, chances are that it's also available in iconvg

Reply:
The icon show is availble in iconvg but the icon hide is not and we must use image png for it

Reply:
You used "ImageRemoveRedEye" as the show icon, but they are both available in iconvg as  ActionVisibility & ActionVisibilityOff.